README.md

```diff
@@ -1,7 +1,43 @@
+# App Configuration Sync action

```

Comment:
>App Configuration Sync [](start = 2, length = 22)

I notice that we're effectively titling the readme here with "App Configuration Sync". I'm wondering if we should also make this the title of the action as specified in action.yml

Reply:
I just looked at the market place page at [https://github.com/marketplace?utf8=%E2%9C%93&type=actions&query=azure](https://github.com/marketplace?utf8=%E2%9C%93&type=actions&query=azure) and it looks like we should change the action.yml title to "Azure App Configuration Sync". 

We should also update this header here to be "Azure App Configuration Sync"

---
In reply to: [354455873](https://github.com/Azure/AppConfiguration-Sync/pull/2#discussion_r354455873) [](ancestors = 354455873)